data/tlg0018/tlg001/__cts__.xml

```diff
@@ -1,7 +1,14 @@
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg0018" xml:lang="grc" urn="urn:cts:greekLit:tlg0018.tlg001">
   <ti:title xml:lang="lat">De Opificio Mundi</ti:title>
-  <ti:edition urn="urn:cts:greekLit:tlg0018.tlg001.opp-grc1" workUrn="urn:cts:greekLit:tlg0018.tlg001" xml:lang="grc">
+  <ti:edition urn="urn:cts:greekLit:tlg0018.tlg001.opp-grc1" workUrn="urn:cts:greekLit:tlg0018.tlg001">
```

Comment:
xml:lang is required here. (See issue from Giuseppe on this)

Reply:
I have added that to my automatic checks!